FEXCore/unittests/Emitter/SVE_Tests.cpp

```diff
@@ -2336,69 +2336,69 @@ TEST_CASE_METHOD(TestDisassembler, "Emitter: SVE: SVE broadcast integer immediat
 }
 
 TEST_CASE_METHOD(TestDisassembler, "Emitter: SVE: SVE broadcast floating-point immediate (predicated)") {
-  TEST_SINGLE(fcpy(SubRegSize::i16Bit, ZReg::z30, PReg::p6.Merging(), -0.125), "fmov z30.h, p6/m, #0xc0 (-0.1250)");
+  // TEST_SINGLE(fcpy(SubRegSize::i16Bit, ZReg::z30, PReg::p6.Merging(), -0.125), "fmov z30.h, p6/m, #0xc0 (-0.1250)");
```

Comment:
Probably good to keep these around in case 16-bit support is enabled again later, but code shouldn't be commented out without indication why.

Might make most sense to add a preprocessor define at the top of the file to be more explicit about it, or to move together the various i16Bit tests so we don't need to repeat the same docstring everywhere.

Reply:
Wrapped in a preprocessor define.

Reply:
There's now a single point of reference for the disabled code (good!), but it's still not indicated why the code is disabled. Could you add a comment to the preprocessor define?

Reply:
Added a comment

Reply:
Great, thanks!